components/AddLiquidity/index.tsx

```diff
@@ -215,7 +217,7 @@ function Component() {
                             }
                         } catch (err) {
                             dispatch(setTxStatusLoading('rejected'))
-                            toast.error(String(err), {
+                            toast.error(t(String(err)), {
```

Comment:
In this case, the err msg from catch will always be 'Rejected by user' that's why I put t(String(err) there. Otherwise, if the err msg is unknown it'll show the original error msg

Reply:
noted - thanks!